scene/gui/label.cpp

```diff
@@ -327,6 +329,16 @@ inline void draw_glyph_outline(const Glyph &p_gl, const RID &p_canvas, const Col
 PackedStringArray Label::get_configuration_warnings() const {
 	PackedStringArray warnings = Control::get_configuration_warnings();
 
+	// FIXME: This is not ideal and the sizing model should be fixed,
+	// but for now we have to warn about this impossible to resolve combination.
+	// See GH-83546.
+	if (!is_inside_tree() || get_tree()->get_edited_scene_root() != this) {
```

Comment:
> `!is_inside_tree`

If it's no in tree`get_parent_control()` will be null, shouldn't it be an opposite or simply `is_part_of_edited_scene()`?

Reply:
This check is only for the root node of the scene and it's needed because `get_tree()` can return null.

Reply:
I mean, I can change it to 

```suggestion
	if (is_inside_tree() && get_tree()->get_edited_scene_root() != this) {
```

Which would simply exclude cases where the node is not inside of a tree. But there is no particular reason to do that.

Reply:
If it's just to ensure `get_tree` is ok, maybe it should be `if (is_inside_tree() && get_tree()->get_edited_scene_root() != this)`. What's the point of generating the warning if it's not in tree.